core/services/assets/I18nRegistry.php

```diff
@@ -231,10 +231,17 @@ private function getJedLocaleData($domain)
     {
         $translations = get_translations_for_domain($domain);
 
+        // check get_user_locale() exists before using it for compatibility with WP4.5
+        if(is_admin() && function_exists('get_user_locale')) {
+            $lang = get_user_locale();
+        } else {
+            $lang = get_locale();
+        }
```

Comment:
We already have a helper in `EEH_DTT_Helper` for this.  So you can simply modify the existing conditional below to:
```
'lang' => is_admin() ? EEH_DTT_Helper::get_user_locale() : get_locale()
```